certmanager/checkupdate.py

```
@@ -0,0 +1,271 @@
+# Function to retrieve certificate data from the database
+def retrieve_certificate_data(partner_id, db_host, db_port, db_user, db_password):
+    try:
+        pms_conn = psycopg2.connect(
+            host=db_host,
+            port=db_port,
+            database="mosip_pms",
+            user=db_user,
+            password=db_password
+        )
+        pms_cursor = pms_conn.cursor()
+        sql_query_cert_alias = f"SELECT certificate_alias FROM pms.partner WHERE id = '{partner_id}';"
```

Ruff (0.16.3)

[error] Possible SQL injection vector through string-based query construction

(S608)